pkg/csi/service/wcp/controller.go

```diff
@@ -541,6 +556,25 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 		log.Error(msg)
 		return nil, err
 	}
+
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.FakeAttach) {
+		fake, err := isFakeAttached(ctx, req.VolumeId)
```

Comment:
isFakeAttached() is doing a query on api server! I think, this will this affect the performance of Detach operation for all non-fake attached volumes. Can we use pvclisters in `isFakeAttached/GetVolumeAnnotations` ?

Reply:
Updated to use pvcLister in GetVolumeAnnotations & UpdateVolumeAnnotations.
```
pvcObj, err := c.informerManager.GetPVCLister().PersistentVolumeClaims(pvcNamespace).Get(pvcName)
```